src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import sys
+
+import requests
+
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def assert_can_communicate_with_server(cromshell_url):
+    """Check Connection with Cromwell Server"""
+
+    try:
+        curl_out = requests.get("{}/api/workflows/v1/backends".format(cromshell_url))
+    except ConnectionError:
+        LOGGER.error("Failed to connect to %s", cromshell_url)
+        sys.exit()
```

Comment:
Does this cause problems in tests?  In java if you system.exit it typically takes out the whole infrastructure.  Shouldn't this just rethrow the exception with some more information and we can decide how to handle uncaught stuff at the top?

Reply:
@jonn-smith ^^

Reply:
replaced all sys.exit() with exception errors